explorer/src/migration/1556119396403-ConvertJobRunSearchableColsToCitext.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class ConvertJobRunSearchableColsToCitext1556119396403
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    await queryRunner.query('CREATE EXTENSION citext;')
+    await queryRunner.query(`
+      DROP INDEX job_run_chainlink_node_id_run_id_idx;
+      DROP INDEX job_id_idx;
+      DROP INDEX job_run_requester_idx;
+      DROP INDEX job_run_request_id_idx;
+      DROP INDEX job_run_tx_hash_idx;
+    `)
+
+    await queryRunner.query(`
+      ALTER TABLE "job_run" ALTER COLUMN "runId" TYPE citext;
```

Comment:
I'm pretty certain this is not compatible with sqlite, you have to do the create new table COPY across trick.

Reply:
@j16r This is reserved just for `explorer` so compatibility with sqlite isn't necessary.

Reply:
SQLite doesn't support `citext` at all.

Reply:
Ahhh! Well carry on then ;)